Formula/hledger.rb

```diff
@@ -1,12 +1,10 @@
 require "language/haskell"
```

Comment:
Is this still needed?

Reply:
You're right, it's not needed since I implemented [MikeMcQuaid's advice](https://github.com/Homebrew/brew/pull/6945#issuecomment-575311420). I've removed it.